content/sensu-go/5.10/guides/scale-event-storage.md

```diff
@@ -0,0 +1,155 @@
+
+- [Why use the Enterprise datastore?](#why-enterprise-datastore)
+- [Prerequisites](#prerequisites)
+- [Configure Postgres](#configure-postgres-for-sensu)
+- [Configure Sensu](#configure-sensu-for-postgres)
+- [Revert to built-in datastore](#revert-to-builtin-datastore)
+
+## Why use the Enterprise datastore? {#why-enterprise-datastore}
+
+For each unique entity/check pair, Sensu records the latest event object in its datastore. By default, Sensu uses the embedded etcd datastore for event storage. The embedded etcd datastore helps you get started, but as the number of entities and checks in your Sensu implementation grows, so does the rate of events being written to the datastore. In a clustered deployment of etcd, whether embedded or external to Sensu, each event received by a member of the cluster must be replicated to other members, increasing network and disk IO utilization. 
+
+Our team documented configuration and testing of Sensu running on bare metal infrastructure in the [sensu/sensu-perf][2] project. This configuration comfortably handled 12,000 Sensu agent connections (and their keepalives) and processed more than 8,500 events per second. 
+
+This rate of events should be more than sufficient for many installations but assumes an ideal scenario where Sensu backend nodes use direct-attached, dedicated non-volatile memory express (NVMe) storage and are connected to a dedicated LAN. Deployments on public cloud providers are not likely to achieve similar results due to sharing both disk and network bandwidth with other tenants. Following the cloud provider's recommended practices may also become a factor because many operators are inclined to deploy a cluster across multiple availability zones. In such a deployment cluster, communication happens over shared WAN links, which are subject to uncontrolled variability in throughput and latency.
+
+Using the Enterprise datastore can help operators achieve much higher rates of event processing and minimize the replication communication between etcd peers. The `sensu-perf` test environment comfortably handles 40,000 Sensu agent connections (and their keepalives) and processes more than 36,000 events per second under ideal conditions. 
+
+## Prerequisites
+
+* Database server running Postgres 9.5 or later
+* Postgres database (or administrative access to create one)
+* Postgres user with permissions to the database (or administrative access to create such a user)
+* [Licensed Sensu Go backend][3]
+
+## Configure Postgres {#configure-postgres-for-sensu}
+
+Before Sensu can start writing events to Postgres, you need a database and an account with permissions to write to that database. To provide consistent event throughput, we strongly recommend exclusively dedicating your Postgres instance to storage of Sensu events.
+
+If you have administrative access to Postgres, you can create the database and user:
+
+``` shell
+$ sudo -u postgres psql
```

Comment:
I believe this is suppose to be `psql postgres`

Reply:
tested on [centos7 with with postgres 9.6.2 ](https://www.linode.com/docs/databases/postgresql/how-to-install-postgresql-relational-databases-on-centos-7/).

Reply:
@Blue0ctober I've just tested your suggestion. It's true, `psql postgres` works, but only if you are the `postgres` user. 

The sudo command I have documented is effectively accomplishing the same thing ("run psql as user postgres") as the Linode instructions you've linked, but through a slightly different path.

As such, I don't know if a change is truly needed here. Did the instructions work for you otherwise?